engine/src/components/details.ts

```diff
@@ -0,0 +1,16 @@
+import Component from "./component";
+
+export default class Details extends Component {
+  getViewModel() {
+    const viewModel = super.getViewModel();
```

Comment:
what's your reasoning behind using `const viewModel = super.getViewModel();` vs `return {...super.getViewModel(), ..}`?

Reply:
I'm fine with either, but lets just stick to one approach for consistency's sake

Reply:
I'll change to destructing